### ISSUE TITLE
IntegrationSpec intermittently times out

### ISSUE THREAD

**Issue by gustavonalle** [ACINQ/eclair#703]
This happens on my 2 cores dev machines and I noticed on travis as well:

```
[ERROR] send an HTLC A->D with an invalid expiry delta for B(fr.acinq.eclair.integration.IntegrationSpec)  Time elapsed: 90.654 s  <<< FAILURE!
java.lang.AssertionError: assertion failed: timeout 90 seconds expired: 
	at fr.acinq.eclair.integration.IntegrationSpec.withFixture(IntegrationSpec.scala:59)
	at fr.acinq.eclair.integration.IntegrationSpec.runTest(IntegrationSpec.scala:59)
	at fr.acinq.eclair.integration.IntegrationSpec.runTests(IntegrationSpec.scala:59)
	at fr.acinq.eclair.integration.IntegrationSpec.org$scalatest$FunSuiteLike$$super$run(IntegrationSpec.scala:59)
	at fr.acinq.eclair.integration.IntegrationSpec.org$scalatest$BeforeAndAfterAll$$super$run(IntegrationSpec.scala:59)
	at fr.acinq.eclair.integration.IntegrationSpec.run(IntegrationSpec.scala:59)
```

**Comment by pm47** [comment]
I caused that in f5a598520551bd5d95efb7cb8a83a37f0a1f7725, one of the tests is unreliable. Will fix ASAP.

**Comment by pm47** [comment]
Fixed with db96c17fd2bb36ecb1705b4c79bf6e85c4052122 and b56f0359fe6aa45674cf716190aacad9227364f4.
